app/src/main/resources/logging.properties

```diff
@@ -0,0 +1,7 @@
+handlers= java.util.logging.FileHandler
+
+# File Logging
+java.util.logging.FileHandler.pattern = ${java.io.tmpdir}/scenebuilder-${version}.log
```

Comment:
Probably better to log to `~/.scenebuilder` (and change the message on the dialog)? It would be more permanent.